hs_core/tests/api/native/test_core_metadata.py

```diff
@@ -1062,20 +1062,12 @@ def test_relation(self):
         self.assertRaises(Exception, lambda: resource.update_metadata_element(self.res.short_id, 'relation',
                                                                               rel_to_update.id, type='isCopiedFrom'))
 
-        # test that duplicate relation types are not allowed
-        self.assertRaises(Exception, lambda: resource.create_metadata_element(
```

Comment:
Seems like a simple check for uniqueness would avoid this. The relations are written to the metadata, so I'm not sure why we would want to allow this. I don't think it will break anything, but I'm just not sure why we would enable duplication.